src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__logo">
+          <a href="#home" class="header__logo-link">
+            <img
+            alt="Logo"
+            class="header__logo-img"
+            src="./images/logo.png">
+          </a>
+        </div>
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#apple" class="header__link is-active">Apple</a>
+            </li>
+            <li class="header__item">
+              <a href="#samsung" class="header__link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="header__item">
              <a href="#apple" class="header__link is-active">Apple</a>
            </li>
            
            <li class="header__item">
              <a href="#samsung" class="header__link">Samsung</a>
            </li>
```

Next time pls add empty line between multiple sibling blocks of html